### ISSUE TITLE
What about embedding size of 32 and learning rate of 0.001 (no decay?)

### ISSUE THREAD

**Issue by kootenpv** [Conchylicultor/DeepQA#9]
Hey, I was wondering why the embedding size is 32? Is that not really small?

Also, I'm more used to seeing a higher learning rate to start with, and a decaying parameter that slowly lowers the learning rate.

Do you have some specific reasons for these values and lack of decay :)?

**Comment by Conchylicultor** [comment]
I choose 32 mainly to fit my hardware limitations but surprisingly, it seems enough to capture the basic semantic. If you computer allows it, feel free to increase this parameter. On the original seq2seq paper, they used an embedding size of 1000. However, they had a much bigger training corpus so I'm not sure this would works well on the current data.

For the learning rate decay, correct me if I'm wrong but it seems to me that the ADAM optimizer used here already adapt the learning rate for the parameters and thus adding a learning rate decay is not necessary. But you can still give it a try, see if it could help.
